internal/store/postgres/appeal_repository_test.go

```diff
@@ -321,9 +321,9 @@ func (s *AppealRepositoryTestSuite) TestFind() {
 			},
 			{
 				filters: &domain.ListAppealsFilter{
-					AccountID: "user@email.com",
```

Comment:
let's also keep the `AccountID` one in the test case

Reply:
done